sample_files/.gitlab-ci.yml

```diff
@@ -0,0 +1,41 @@
+image: vauxoo/odoo-80-image-shippable-auto
+stages:
+  - test
+
+variables:
+    PSQL_VERSION: "9.5"
+    VERSION: "11.0"
+    ODOO_REPO: "odoo/odoo"
+    ODOO_BRANCH: "11.0"
+    TRAVIS_BUILD_DIR: "$CI_PROJECT_DIR"
+
+lint:
+  stage: test
+  tags:
+    - lint
+  variables:
+    LINT_CHECK: "1"
+    TESTS: "0"
+  script:
+    - sudo apt-get update && sudo apt-get install -y dos2unix
+    - git clone https://github.com/vauxoo/maintainer-quality-tools.git -b master ${HOME}/maintainer-quality-tools
```

Comment:
IMO this should be the OCA repo instead of Vauxoo fork

Reply:
Good point, thanks
(copy&paste issue)